Marlin/UltiLCD2_menu_material.cpp

```diff
@@ -50,13 +51,16 @@ static void lcd_menu_material_temperature_settings();
 static void lcd_menu_material_retraction_settings();
 static void lcd_menu_material_retraction_settings_per_nozzle();
 static void lcd_menu_material_settings_store();
-bool lcd_material_check_temperature(unsigned long temperature);
-bool lcd_material_check_bed_temperature(unsigned long temperature);
-bool lcd_material_check_fan_speed(unsigned long fanspeed);
-bool lcd_material_check_material_flow(unsigned long flow);
-bool lcd_material_check_material_diameter(double diameter);
-bool lcd_material_check_retraction_length(float length);
-bool lcd_material_check_retraction_speed(unsigned long speed);
+static bool hasInvalidNozzleTemperature(uint16_t temperature);
```

Comment:
~Suggest using 'hasValid...' to (possibly) prevent 'double negative' comparison.
```if not hasInvalid....:```~
Checcking is done the other way around, comment is moot.